LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Lorna Kaboro
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
```

Comment:
- [ ] Dear @Lornakaboro, I'm really sorry to make this a required change again, But Please remember it's not allowed to have a broken link in our readme, special the license file. I can see you have added the file, but the link is till broken.  Please use `[MIT](./LICENSE)` instead of `[MIT](./LICENSE.md)` I have noticed removing the `.md` extension from the link makes it work. 

**Please also check your changes online before submitting for another review**